File: DownloadFiles/TGNet/Chapter04/VB/Exercise4-3/MileageCalculator/MileageCalculator/Form1.vb
Public Class Form1

  Private Sub btnGetMileage_Click _
  (ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnGetMileage.Click
    'Call the GetAnswer Method
    GetAnswer()
    MessageBox.Show("Done")
  End Sub

  'Create a Method to calculate Mileage
  Public Sub GetAnswer()

    'Create 3 variables to work with
    Dim Mileage, Distance, Gallons As Decimal

    'Add code to set these values from the UI
    Distance = Convert.ToDecimal(txtDistance.Text)
    Gallons = Convert.ToDecimal(txtGallons.Text)
    Mileage = Distance / Gallons

    MessageBox.Show(Mileage.ToString, "Answer")
  End Sub

End Class

File: DownloadFiles/TGNet/Chapter04/C#/Exercise4-3/MileageCalculator/MileageCalculator/Form1.cs
#region Using directives

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Windows.Forms;

#endregion

namespace MileageCalculator
{
    partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void btnGetMileage_Click(object sender, EventArgs e)
        {
            GetAnswer();
            MessageBox.Show("Done");
        }
    
       //Create a Method to calculate Mileage
       public void  GetAnswer()
        {
            //Create 3 variables to work with
            Decimal Mileage, Distance, Gallons;
            //Add code to set these values from the UI
            Distance = Convert.ToDecimal(txtDistance.Text);
            Gallons = Convert.ToDecimal(txtGallons.Text);
            Mileage = Distance / Gallons;

            MessageBox.Show(Mileage.ToString(), "Answer");
         }
    } //End of Class
}